todo-app/src/components/addingContainer.js

```diff
@@ -0,0 +1,43 @@
+
+import React, { Component } from "react";
+import FaPlus from "react-icons/lib/fa/plus"
+
+
+export default class AddingContainer extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            description: "",
+            deadLine: ""
+        }
+    }
+
+    handleDescriptionChange = (event) => {
+        this.setState({ description: event.target.value })
+    }
+    handleDeadlineChange = (event) => {
+        this.setState({ deadLine: event.target.value })
+
+    }
+    handleCreating = () => {
+        let descriptionInput = this.state.description
+        let deadLineInput = this.state.deadLine
+         this.props.create(descriptionInput, deadLineInput)
```

Comment:
Again a small naming issue, please rename `create` to something a little bit more descriptive...